app/views/layouts/body/_navbar.html.erb

```diff
@@ -1,18 +1,27 @@
 <div class="navbar navbar-default">
   <div class="container">
     <div class="navbar-header">
-      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+      <button type="button" class="navbar-toggle" data-toggle="collapse"
+        data-target=".navbar-collapse">
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
     </div>
     <div class="navbar-collapse collapse">
       <ul class="nav navbar-nav">
-        <li><%= link_to "Home", home_path %></li>
-        <li><%= link_to "Browse", browse_path %></li>
-        <li><%= link_to "Search", search_path %></li>
-        <li><%= link_to "About", about_path %></li>
+        <li class="<%= active?(home_path) %>">
+          <%= link_to "Home", home_path %>
+        </li>
+        <li class="<%= active?(/^\/browse/, request.path) %>">
```

Comment:
Note:  I believe these will break if running the app at a sub-uri (which hasn't happened yet, but something to keep in mind)

Reply:
Good point! I hadn't thought of that. That may be a reason to go with the `:action` comparison instead 🤔

Reply:
:/  our yesterday versions lost the battle of forward thinking

Reply:
Almost